pype/plugins/global/publish/extract_review.py

```diff
@@ -82,6 +82,12 @@ def process(self, instance):
                 repre_new = repre.copy()
                 ext = profile.get("ext", None)
                 p_tags = profile.get('tags', [])
+
+                # append repre tags into profile tags
+                for t in tags:
+                    if t not in p_tags:
+                        p_tags.append(t)
```

Comment:
This doesn't make sence. You're adding tags from profile to representation's tags before filtering and processing? So representations which shouldn't be even processed, are, just because profile contain tag which match tags filters in process part?

Reply:
the reason is that the ability to add tags in `ExtractReviewCutUpVideo` has been added on `self.tags_addition` so production can per project add some additional tags on re-edited review videos. This way the process merging those tags into preset's profile tags

Reply:
Tags in profile are not for filtering, but for tagging with those functions. merging them together with repre tags actually makes sense.